setup.cfg

```diff
@@ -0,0 +1,5 @@
+[flake8]
+# W391 blank line at end of file
+# E302 expected 2 blank lines, found 1
+ignore = W391,E302
```

Comment:
@QuLogic @nirbheek removed.

@jpakkane it's not only about code-style, but also about checks for unitialized variables and such (something what me and @TingPing were fixing recently), does clang-format fix the code? ;)

After all, we can tune warnings.